content/pt-br/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -0,0 +1,872 @@
+verificar se um contêiner está sendo terminado por chegar no limite de algum
+recurso, utilize o comando `kubectl describe pod` no Pod em questão:
+
+```shell
+kubectl describe pod simmemleak-hra99
+```
+
+A saída será semelhante a:
+```
+Name:                           simmemleak-hra99
+Namespace:                      default
+Image(s):                       saadali/simmemleak
+Node:                           kubernetes-node-tf0f/10.240.216.66
+Labels:                         name=simmemleak
+Status:                         Running
+Reason:
+Message:
+IP:                             10.244.2.75
+Containers:
+  simmemleak:
+    Image:  saadali/simmemleak:latest
+    Limits:
+      cpu:          100m
+      memory:       50Mi
+    State:          Running
+      Started:      Tue, 07 Jul 2019 12:54:41 -0700
+    Last State:     Terminated
+      Reason:       OOMKilled
+      Exit Code:    137
+      Started:      Fri, 07 Jul 2019 12:54:30 -0700
+      Finished:     Fri, 07 Jul 2019 12:54:33 -0700
+    Ready:          False
+    Restart Count:  5
+Conditions:
+  Type      Status
+  Ready     False
+Events:
+  Type    Reason     Age   From               Message
+  ----    ------     ----  ----               -------
+  Normal  Scheduled  42s   default-scheduler  Successfully assigned simmemleak-hra99 to kubernetes-node-tf0f
+  Normal  Pulled     41s   kubelet            Container image "saadali/simmemleak:latest" already present on machine
+  Normal  Created    41s   kubelet            Created container simmemleak
+  Normal  Started    40s   kubelet            Started container simmemleak
+  Normal  Killing    32s   kubelet            Killing container with id ead3fb35-5cf5-44ed-9ae1-488115be66c6: Need to kill Pod
+```
+
+No exemplo acima, o campo `Restart Count:  5` indica que o contêiner `simmemleak`
+deste Pod foi terminado e reiniciado cinco vezes até o momento. A razão
+`OOMKilled` demonstra que o contêiner tentou consumir mais memória do que o seu
+limite.
+
+O próximo passo neste cenário seria vasculhar e depurar o código da aplicação,
+procurando por vazamentos de memória. Se você determinar que a aplicação está se
+comportando conforme o esperado, considere aumentar o limite (e possivelmente
+o requerimento) de memória para aquele contêiner.
+
+## {{% heading "whatsnext" %}}
+
+* Pratique [a criação de requerimentos de recursos de memória em contêineres e Pods](/docs/tasks/configure-pod-container/assign-memory-resource/).
+* Pratique [a criação de requerimentos de CPU em contêineres and Pods](/docs/tasks/configure-pod-container/assign-cpu-resource/).
+* Leia como a referência da API define um [contêiner](/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container)
+  e seus [requerimentos de recursos](/docs/reference/kubernetes-api/workload-resources/pod-v1/#resources).
+* Leia sobre [quotas de projeto](https://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html) no XFS.
```

Comment:
This link is currently broken. I found a manual page [here](https://man7.org/linux/man-pages/man8/xfs_quota.8.html). Can I replace the broken link for the manual page in this context?

Reply:
Maybe you can open a PR/issue to fix the original docs and use the manual page here, I didn't see a problem with this approach :)

Reply:
Will open an issue to discuss the best way forward to fix the link!

Reply:
It looks like someone already fixed the link. I will copy the new link to the translation here.